Update sites-filter.js

carlj11 opened the pull request:
Add condition &sort_by=created&sort_order=desc (L100) to display most recent exhibit first on the exhibit landing page. This change is for the item_set sort oder but let's see if this actually changes the Site sort order on the main home page. Default behavior is to display the oldest (by date) created Site first.

## Developer

Test setting exhibit Site sort order by created&sort_order=desc date to render most recently created exhibit Sites first.

orig:
- `let url = '/api/items?item_set_id=${id}'`

now:
- `let url = '/api/items?item_set_id=${id}&sort_by=created&sort_order=desc'`

Write a brief summary of changes here, but do not repeat commit messages (which
should have a more complete description).


### Tickets affected

- https://mitlibraries.atlassian.net/browse/ES-3204

- https://mitlibraries.atlassian.net/browse/IN-1327

### Version (see config/theme.ini)

- [ X] The theme's version number has been incremented, setting up a production
      deploy.

Version number has now been updated to 1.0.1.   NOTE: at this point we're not ready for a production deploy but this may help with debugging some issues with the automated update process.

### Documentation

- [ ] Project documentation has been updated.
- [ ] No documentation changes are needed.

### Accessibility

- [ ] ANDI or WAVE has been run in accordance to [our guide](https://mitlibraries.github.io/guides/basics/a11y.html).
- [ ] New flagged issues not already resolved in this PR have been ticketed 
      (link in the Pull Request details above).
- [ ] This PR contains no changes to the view layer.

### Stakeholder approval

- [ ] Stakeholder approval has been confirmed (see ticket).
- [ ] Stakeholder approval will happen retroactively.
- [ ] Stakeholder approval is not needed.

### Dependencies

- [ ] New dependencies have been added
- [ ] Dependencies have been updated
- [ ] No dependencies have changed

### Additional context needed to review

E.g., if the PR includes updated dependencies and/or data
migration, or how to confirm the feature is working.

NOTE: 6/27 I tried running the 'make update' step (didn't realize I could run prior to submitting this PR) which is supposed to update the favicon but that failed. See our JIRA ticket: https://mitlibraries.atlassian.net/browse/IN-1327


## Code Reviewer

##### Code

- [x] I have confirmed that the code works as intended.
- [x] Any CodeClimate issues have been fixed or confirmed as
      added technical debt.
- [x] New dependencies are appropriate or there were no changes.

##### Documentation

- [ ] The commit message is clear and follows our guidelines
      (not just this pull request message).
- [x] The documentation has been updated or is unnecessary.

carlj11 committed:
Update sites-filter.js

Add condition &sort_by=created&sort_order=desc (L100) to display most recent exhibit first on the exhibit landing page. This change is for the item_set sort oder but let's see if this actually changes the Site sort order on the main home page. Default behavior is to display the oldest (by date) created Site first.

matt-bernhardt commented:
I've run the workflow to deploy this branch to staging, and am trying to confirm that things work as expected. The review is under way, and I'm hoping to have something meaningful to say tomorrow.

Deploy workflow result: https://github.com/MITLibraries/mitlibraries-theme-omeka/actions/runs/15887300608/job/44802306845

matt-bernhardt commented:
I'm not seeing an update to the deployed `sites-filter.js` file, like I'd expect to see during a deploy. Compare the deployed file:

https://mitlibraries-stage.omeka.net/themes/mitlibraries-theme-omeka/asset/js/sites-filter.js?v=1.0

with the version in this PR:

https://github.com/MITLibraries/mitlibraries-theme-omeka/pull/62/files#diff-a077944fb4f41f41929c706212f6703837423cea26c34f6ce37b940104ed2f04

My expectation was that the change near the bottom of the file would be reflected, but I don't see it yet.

As a second check, Carl could you please try bumping the version number of the theme by incrementing this line from `1.0` to `1.0.1`? That would give us something else to look at that would be an outcome of the deploy process. 

https://github.com/MITLibraries/mitlibraries-theme-omeka/blob/main/config/theme.ini#L3

matt-bernhardt commented:
On a related note, I don't see in this PR a change to the favicon yet - has this been added, or is that still on the to-do list? Running `make update` from the theme root should result in a change to `asset/img/mitlib-style/favicon.ico`

carlj11 committed:
Update _layouts.scss

(carlj, 2025-06-27) After Matt B. consult, add small snippet of styles to remove unwanted horizontal grey bar. Necessitated by updated Block behavior by omeka.org/DS in Omeka version 4.1.1.

DS says the rule in. question is:

.page-layout-normal .block {
   margin: 1rem 0;
}

carlj11 committed:
Update theme.ini

Increment version number to 1.0.1

carlj11 committed:
Update favicon only

carlj11 committed:
Update _layouts.scss

Add snippet of styles to remove unwanted horizontal grey bar. Make everything inside the .wrap-page appear on a white background  to prevent the body background color from bleeding through

carlj11 committed:
Remove unwanted horizontal grey bar near top.
Indent snippet to make everything inside .wrap-page appear on a white background.

carlj11 committed:
Merge branch 'carlj_update_exhibit_display_order_and_favicon' of https://github.com/MITLibraries/mitlibraries-theme-omeka into carlj_update_exhibit_display_order_and_favicon

carlj11 committed:
compiled stylesheet with gulp css

matt-bernhardt reviewed (commented):
This is looking good to me in the abstract, but I'm going to hold off on approving it formally until we get it deployed to staging to confirm that everything looks okay in action.

carlj11 committed:
Remove extraneous descriptive text in _layouts.scss; stylelint version in package.json; package-lock.json integrity sha512- value update(?)

Remove verbose reference to the date of my changes to the file in _layouts.scss

We don't fully know what the changes to pakcage-lock.json and package.json mean. Changes to integrity sha512-  and stylelint version package.json. Matt B. didn't think whatever warranted the commit was significant but to go ahead and commit anyway. We didn't ourselves touch the files.

matt-bernhardt reviewed (approved):
This all looks good to me, and when I deploy it to the staging tier it seems to work as expected.

We'll need a follow-up ticket to update the theme to load some assets from our CDN rather than package them with the theme itself (which should also include some markup changes to how the favicon gets loaded, which I've described in ES-3204) - but for now, it seems like Stephanie is okay with what's here.

I'm comfortable with merging this as-is, but if you have concerns about the sort order you mentioned in the ticket, we can talk through those before merging.
